### PR TITLE
Allow pylint_bin to always find realpath of pip installs

### PR DESCRIPTION
**Pull request by fejta**
ref #13140 #13145

### CODE REVIEW

File: hack/pylint_bin.py [hunk 1]
```diff
@@ -26,11 +26,28 @@
...
             full = os.path.join(path, something)
             real = os.path.realpath(full)
-            if real != full:
-                EXTRAS.add(os.path.dirname(real))
-                break
+            # If we use pip_import() then there will be
+            # a WHEEL file symlink we can use to find the real path.
+            # TODO(fejta): use pip exclusively
+            # (ruamel has C code, which doesn't work with pip_import())
+            # yet work) but this appears to be
```
Comment (michelle192837) [inline hack/pylint_bin.py]: 'yet work'?

Comment (fejta) [reply hack/pylint_bin.py]: Done

**Pull request merged**